Resources/Help/analyses/ancovabayesian.md

```diff
@@ -123,22 +122,22 @@ To create a descriptive plot, place the independent variable on the horizontal a
 - BF<sub>inclusion</sub> : This column contains the change from prior inclusion odds to posterior inclusion odds for each component averaged by all the models the component is in. 
 
 #### Model Averaged Posterior Summary 
-- Variable: This column contains all the variables and interactions included in the models. 
-- Level: The levels/groups of the variables and interactions included in the model. 
-- Mean: The mean 
-- SD: The standard deviation. 
-- % Credible Interval: The credible interval of the mean. By default, this is set to 95%. 
+- Variable: This column contains all the fixed factors, their interactions, and covariates included in the models. The first row contains information about the intercept. 
+- Level: Each level of the factor and combination of levels of the interactions that are included in the model. 
+- Mean: The model averaged mean. For the factors, this is the deviation from the intercept for each level of the factor. Together, the level means for a factor are equal to zero.  
+- SD: The standard deviation of the model averaged mean. 
+- % Credible interval: The credible interval of the mean. By default, this is set to 95%. 
   - Lower: The lower bound of the credible interval of the mean. 
   - Upper: The upper bound of the credible interval of the mean. 
 
 #### Model Averaged Posterior Distributions 
-For each factor, and interaction the posterior distributions per level is displayed, with on the x-axis the independent variable and on the y-axis the density. The posterior distribution for each level can either be displayed in the same plot, or by different plots for each level. 
+For each factor, interaction, and covariate the model averaged posterior distributions per level is displayed, with on the x-axis the factor and on the y-axis the density. The posterior distribution for each level can either be displayed in the same plot, or by different plots for each level. 
```

Comment:
"For each factor, interaction, and covariate, the model averaged posterior distributions per level are displayed, ..."